.eslintrc.js

```diff
@@ -42,6 +42,13 @@ module.exports = {
 		'@wordpress/dependency-group': 'error',
 		'@wordpress/gutenberg-phase': 'error',
 		'@wordpress/react-no-unsafe-timeout': 'error',
+		'@wordpress/i18n-text-domain': [
+			'error',
+			{
+				allowDefault: true,
```

Comment:
Do you consider there to be any meaningful difference between:

```
'@wordpress/i18n-text-domain': [ 'error', { allowDefault: true } ]
```

...and:

```
'@wordpress/i18n-text-domain': 'off'
```

In other words, do we need an option, if the intent is to exempt the project from the behavior this rule otherwise enforces?

Reply:
With `'@wordpress/i18n-text-domain': 'off'` you would not catch cases like using a variable as a text domain, or using text domain `'gutenberg'` when in fact it should be the default text domain.